pkg/pillar/cmd/msrv/msrv.go

```diff
@@ -549,6 +573,33 @@ func (msrv *Msrv) Run(ctx context.Context) (err error) {
 		return err
 	}
 
+	// Wait for the EdgeNodeInfo, and EdgeNodeCert and ControllerCert publications
+	var edgenodeCertInitialized, controllerCertInitialized, edgenodeInfoInitialized bool
+	for !edgenodeCertInitialized || !controllerCertInitialized || !edgenodeInfoInitialized {
```

Comment:
Not all of the endpoints need those subscriptions to operate, but I believe that this wait won't be noticed by end clients

Reply:
yeh, assume it is normal a restart of the eve device, those things are part of the cached data in /persist, so it should be fast.